test/robust_pmap.jl

```diff
@@ -10,25 +10,35 @@
                 i += 1
                 throw(err)
             end
-            isodd(x)
+            return isodd(x)
         end
         return throw_isodd
     end
 
     # Check other errors don't retry
     throw_isodd = make_throw_isodd(ErrorException("Error"))
-    @test_throws ErrorException robust_pmap(throw_isodd, input)
+    if VERSION < v"1.8-"
+        @test_throws ErrorException robust_pmap(throw_isodd, input)
+    else
+        @test_throws "Error" robust_pmap(throw_isodd, input)
+    end
```

Comment:
if we are testing for presents of the word Error then we might as well just

```suggestion
    @test_throws Exception robust_pmap(throw_isodd, input)
```

Reply:
Well, just since the string message is `Error` here... could change it to `ErrorException("Error123")` and test for "Error123"

Reply:
sounds good